### DISCUSSION TITLE
如何實現空格上屏而非ENTER上屏？

### DISCUSSION THREAD

**Discussion by chakkkkwan** [rime/home#984] [Q&A]
如題

主要是打英文單字的時候，不像Mac一樣用空格選字。

p.s. 倉頡使用者

**Comment by lotem** [comment]
不支持英文。
輸入法不懂英文。
用倉頡打中文用空格上屏。用回車鍵上屏鍵入的字符。

**Comment by rtransformation** [comment]
你安一个基本包含日常使用英文的大点儿的英文词库，然后追加到你现在输入法里，作为副翻译器就行了。
